packages/bpk-component-autosuggest/src/BpkAutosuggest-test.js

```diff
@@ -64,23 +69,24 @@ describe('BpkAutosuggest', () => {
 
   it('should set the input reference', () => {
     let inputRef;
-    const storeAutosuggestReference = autosuggest => {
-      if (autosuggest !== null) {
-        inputRef = autosuggest.input;
-      }
+
+    const storeAutosuggestReference = ref => {
+      inputRef = ref;
     };
+
     const tree = ReactTestUtils.renderIntoDocument(
       <BpkAutosuggest
         suggestions={suggestions}
         onSuggestionsFetchRequested={onSuggestionsFetchRequested}
         onSuggestionsClearRequested={onSuggestionsClearRequested}
         getSuggestionValue={getSuggestionValue}
         renderSuggestion={renderSuggestion}
-        inputProps={inputProps}
-        ref={storeAutosuggestReference}
+        inputProps={{ inputRef: storeAutosuggestReference, ...inputProps }}
```

Comment:
is it worth update the readme/docs with the usage of the inputProps?

Reply:
The docs are here -> https://backpack.github.io/components/web/autosuggest

What would you update in particular?

Reply:
nevermind, it's all covered!